vm-setup/roles/v1aX_integration_test/tasks/healthcheck.yml

```diff
@@ -0,0 +1,208 @@
+# Worker CAPI healthcheck and remediation controller test
+
+- name: Deploy CAPI healthcheck for worker
+  shell: kubectl apply -f "{{ CRS_PATH }}/machine-healthcheck-worker.yaml" -n "{{ NAMESPACE }}"
+
+- name: Verify that CAPI healthcheck is created
+  kubernetes.core.k8s_info:
+    api_version: cluster.x-k8s.io/v1beta1
+    kind: MachineHealthCheck
+    namespace: "{{ NAMESPACE }}"
+  retries: 20
+  delay: 20
+  register: capi_healthcheck
+  until: (capi_healthcheck is succeeded) and
+         (capi_healthcheck.resources | length == 1)
+
+- name: Fetch the target cluster kubeconfig
+  k8s_info:
+    kind: secrets
+    name: "{{ CLUSTER_NAME }}-kubeconfig"
+    namespace: "{{ NAMESPACE }}"
+  register: metal3_kubeconfig
+
+- name: Decode and save cluster kubeconfig
+  copy:
+    content: "{{ metal3_kubeconfig.resources[0].data.value | b64decode }}"
+    dest: "/tmp/kubeconfig-{{ CLUSTER_NAME }}.yaml"
+
+# - name: Install Calico to workflow cluster
+#   shell: kubectl --kubeconfig /tmp/kubeconfig-{{ CLUSTER_NAME }}.yaml apply -f https://docs.projectcalico.org/v3.20/manifests/calico.yaml
+
+# Install Calico
+- name: Download Calico manifest
+  get_url:
+    url: https://docs.projectcalico.org/manifests/calico.yaml
+    dest: /tmp/
+    mode: '664'
+  register: calico_manifest
+
+- name: Apply Calico manifest
+  kubernetes.core.k8s:
+    state: present
+    src: "/tmp/calico.yaml"
+    kubeconfig: "/tmp/kubeconfig-{{ CLUSTER_NAME }}.yaml"
+  register: install_cni
+
```

Comment:
We are already doing the same tasks as part of [verify.yml ](https://github.com/metal3-io/metal3-dev-env/blob/master/vm-setup/roles/v1aX_integration_test/tasks/verify.yml#L7-L71). Are we not running verify before this task? I guess we are, if we are not, we should to remove code duplication. Also, in metal3-dev-env we [pin calico ](https://github.com/metal3-io/metal3-dev-env/blob/master/vm-setup/roles/v1aX_integration_test/tasks/verify.yml#L22) to a specific version to avoid docker pull rate limit issue. We dont download latest.

Reply:
Verify action was not added here before. Working on it.